src/classes/GE_GiftEntryController_TEST.cls

```diff
@@ -34,15 +34,15 @@
 * @group-content ../../ApexDocContent/GiftEntry.htm
 * @description Unit tests to cover methods in GE_GiftEntryController.
 */
-@isTest
+@IsTest
```

Comment:
I read the documentation as suggesting that this keyword begin with a lowercase "i".  Is there a different suggestion elsewhere? @voduyemi 

https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_annotation_isTest.htm

Reply:
No, not that I know of, I was following the IlluminatedCloud's suggestion. But we should align with the documentation. I'll make the change.

Reply:
Updated!

Reply:
Thank you!